gwt-event-dom/pom.xml

```diff
@@ -68,7 +73,7 @@
         <version>${maven.gwt.plugin}</version>
         <extensions>true</extensions>
         <configuration>
-          <moduleName>org.gwtproject.event.EventDOM</moduleName>
+          <moduleName>org.gwtproject.event.dom.DomEvent</moduleName>
```

Comment:
That fixes the bug! 

By updating this property of the plugin, we don't need to rename and move the module.gwt.xml. So, the next change is not necessary. (Was not able to add a comment there.